src/types/hull-segments-changes.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+
+import { HullSegmentType } from "./";
+
+/**
+ * Represents segment changes in UserChangesType
+ */
+export type HullSegmentsChangesType = {
```

Comment:
I would suggest to label it `HullSegmentChangesType`, changes is already plural, so i think we should stick to segment in singular form.